netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -199,11 +203,12 @@ protected void onStreamError(ChannelHandlerContext ctx, Throwable cause,
       StreamException http2Ex) {
     logger.log(Level.WARNING, "Stream Error", cause);
     Http2Stream stream = connection().stream(Http2Exception.streamId(http2Ex));
-    if (stream != null) {
+    NettyServerStream serverStream = stream != null ? serverStream(stream) : null;
```

Comment:
It actually wasn't necessary ... I was verifying the content-type too early, before the stream was set.  Reverted.